static/js/controllers/contacts-edit.js

```diff
@@ -34,7 +34,7 @@ var _ = require('underscore');
         return UserDistrict()
           .then(function(facilityId) {
             if (!facilityId) {
-              return $q.resolve();
+              return $q.reject('Facility ID not supplied');
```

Comment:
I just made a change to that file! For similar reasons, I was dealing with a bug for admin user.
UserDistrict returns empty string for admin, it doesn't throw, because admin doesn't have a facility. 
So this change will break admin.